src/main/java/ua/chernonog/onlinebookstore/dto/request/CreateBookRequestDto.java

```diff
@@ -17,7 +17,7 @@ public class CreateBookRequestDto {
     @NotEmpty(message = "Author cannot be empty")
     private String author;
     @NotEmpty(message = "ISBN cannot be empty")
-    @Pattern(regexp = "^(?:\\d{9}[\\dX]|97[89]\\d{10})$", message = "Invalid ISBN format")
+    @Pattern(regexp = "(?:978|979)?\\d{9}[\\dX]", message = "Invalid ISBN format")
     private String isbn;
```

Comment:
There is a ready-made validator based on the @ISBN annotation, but I think it's ok to use the simplified validation here

Reply:
Please consider following annotation